### DISCUSSION TITLE
Projector mode

### DISCUSSION THREAD

**Discussion by ubavic** [OpenHantek/OpenHantek6022#389] [Ideas]
I frequently use OpenHantek for my electronics course, showcasing it on a screen via a projector. Although the audience can generally see the crucial signal characteristics, the visibility is hindered by the thinness of the lines. I recalled that the Desmos graphing calculator offers a projector mode (now called [Enlarge Display](https://help.desmos.com/hc/en-us/articles/4404860698253-What-Accessibility-features-does-Desmos-offer-)), enhancing line thickness and digit size for better visibility. I believe implementing a similar feature in OpenHantek would be good for talks...

Would the program's author be interested in incorporating such a feature? If so, how feasible would it be for a first-time contributor but experienced programmer (like myself) to undertake this implementation?

**Comment by Ho-Ro** [comment]
Sorry for my late response. I remember trying to increase the trace thickness some time ago, without success. If you have experience with OpenGL, I would encourage you to try your luck - I'm open to any improvements.
